src/Message.php

```diff
@@ -20,155 +25,227 @@
-     * @param string $value The body content in the message component.
-     * Message widget will also be treated as the body content, and will be rendered before this.
+     * @param array $values Attribute values indexed by attribute names.
      *
      * @return self
+     *
+     * See {@see \Yiisoft\Html\Html::renderTagAttributes()} for details on how attributes are being rendered.
      */
-    public function body(string $value): self
+    public function attributes(array $values): self
     {
         $new = clone $this;
-        $new->body = $value;
+        $new->attributes = $values;
         return $new;
     }
 
     /**
-     * Returns a new instance with the specified color header message.
+     * Returns a new instance with the specified prefix to the automatically generated widget IDs.
      *
-     * @param string $value Setting default 'is-dark', 'is-primary', 'is-link', 'is-info', 'is-success', 'is-warning',
-     * 'is-danger'.
+     * @param string $value The prefix to the automatically generated widget IDs.
      *
      * @return self
      */
-    public function headerColor(string $value): self
+    public function autoIdPrefix(string $value): self
     {
         $new = clone $this;
-        $new->headerColor = $value;
+        $new->autoIdPrefix = $value;
         return $new;
     }
 
     /**
-     * Returns a new instance with the specified header message.
+     * The body content in the message component. Message widget will also be treated as the body content, and will be
+     * rendered before this.
      *
-     * @param string $value The header message in the message component.
-     * Message widget will also be treated as the header content, and will be rendered before body.
+     * @param string $value The body content in the message component.
      *
      * @return self
      */
-    public function headerMessage(string $value): self
+    public function body(string $value): self
     {
         $new = clone $this;
-        $new->headerMessage = $value;
+        $new->body = $value;
         return $new;
     }
 
     /**
-     * Returns a new instance with the specified HTML attributes for the widget container tag.
+     * The HTML attributes for the widget body tag.
+     *
+     * @param array $values Attribute values indexed by attribute names.
      *
-     * @param array $value The HTML attributes for the widget container tag.
+     * @return self
      *
      * {@see Html::renderTagAttributes()} for details on how attributes are being rendered.
+     */
+    public function bodyAttributes(array $values): self
+    {
+        $new = clone $this;
+        $new->bodyAttributes = $values;
+        return $new;
+    }
+
+    /**
+     * The attributes for rendering the close button tag.
+     *
+     * The close button is displayed in the header of the modal window. Clicking on the button will hide the modal
+     * window. If {@see unclosedButton} is false, no close button will be rendered.
```

Comment:
"Unclosed button" doesn't fit. I'd name it like `withoutCloseButton()`

Reply:
Also, it can't be `false` since the method does not currently accept a boolean value.

Reply:
done.